coalib/misc/Constants.py

```diff
@@ -25,7 +25,7 @@
                             'coala or running `pip3 install "%s"`. Be aware '
                             'that the latter solution might break other '
                             'python packages that depend on the currently '
-                            'installed version.')
+                            'installed version. The "%s" is conflicting')
```

Comment:
where is this `%s` replaced?

Reply:
It's not replaced, Its added

Reply:
Can you tell me why the tests are failing

Reply:
Where are they replaced?

Reply:
the tests are not failing, the coverage is not 100% achieved ;P

Reply:
no actually tests are failing as well Look at
https://circleci.com/gh/coala/coala/15749?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
They are replaced coalib/misc/Exception.py in the function get_exitcode, I have committed to that file as well you can have a look.

Reply:
You are, but you are replacing it with `Bear.name()`, which isnt *the* bears name.